tracing/src/main/java/com/palantir/remoting2/tracing/Tracers.java

```diff
@@ -96,11 +100,14 @@ public static Runnable wrap(Runnable delegate) {
         @Override
         public V call() throws Exception {
             Trace originalTrace = Tracer.copyTrace();
+            String originalMdcTraceIdValue = MDC.get(MDC_KEY);
             Tracer.setTrace(trace);
+            MDC.put(MDC_KEY, trace.getTraceId());
```

Comment:
@uschi2000 we may want to think about a non-MDC mechanism for inclusion of traceId (and userId, sessionId, or others) in logs, thoughts on using our custom layout to do this instead of this approach?